build.sbt

```diff
@@ -35,4 +35,4 @@ initialize := {
 }
 
 
-Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-u", sys.env.getOrElse("SBT_JUNIT_OUTPUT", "junit"))
+Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-u", sys.env.getOrElse("SBT_JUNIT_OUTPUT", "junit"), "-o")
```

Comment:
sorry a quick question - in buid.yml the test folder is mentioned as`junit-test`and here its `junit`, do we need to change in build.yml or here to keep same name?
https://github.com/guardian/editors-picks-uploader/blob/eaf510522e24b9e90815164d5021c6f0991a5d00/.github/workflows/build.yml#L37

Reply:
ah yes thats the case (my bad - i missed checking in sbt_junit_output folder name). go ahead 👍